module/canvas/ability-template.mjs

```diff
@@ -180,7 +181,8 @@ export default class AbilityTemplate extends MeasuredTemplate {
    */
   async _onConfirmPlacement(event) {
     await this._finishPlacement(event);
-    const destination = canvas.grid.getSnappedPosition(this.document.x, this.document.y, 2);
+    let interval = canvas.grid.type === CONST.GRID_TYPES.GRIDLESS ? 0 : 2;
```

Comment:
```suggestion
    const interval = canvas.grid.type === CONST.GRID_TYPES.GRIDLESS ? 0 : 2;
```

Reply:
In the `_onRotatePlacement` method, should the `delta` and `snap` variables be instead declared with `const`, as well? I figured there was some reason they are declared with `let` and went with that to keep consistency.

Reply:
Yeah, anything that isn't redefined later should be a `const`, so you can change those and also `now` in `_onMovePlacement`.